### PR TITLE
Fixing editorial inconsistencies

### PR DESCRIPTION
**Pull request by vkay11**
Improved and removed inaccuracies and inconsistencies from several markdown files.

- [x] Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you explained what your changes do, and why they add value to the Guides?

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

-----

### CODE REVIEW

File: _articles/best-practices.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ For example, @lord discovered that having a project vision helped him figure out
 
 <aside markdown="1" class="pquote">
   <img src="https://avatars.githubusercontent.com/lord?s=180" class="pquote-avatar" alt="avatar">
-  I fumbled it. I didn't put in the effort to come up with a complete solution. Instead of an half-assed solution, I wish I had said "I don't have time for this right now, but I'll add it to the long term nice-to-have list."
```
Comment (MaryJaneLove4) [inline _articles/best-practices.md:50]: a

**Review by MaryJaneLove4 — APPROVED** [review]

**Comment by MaryJaneLove4** [comment]
Ok

**Comment by MaryJaneLove4** [comment]
E?

**Review by MaryJaneLove4 — APPROVED** [review]
Ok

**Review by MaryJaneLove4 — APPROVED** [review]

File: _articles/getting-paid.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ related:
 
 ## Why some people seek financial support
 
-Much of open source work is volunteered. For example, someone might come across a bug in a project they use and submit a quick fix, or they might enjoy tinkering with an open source project in their spare time.
+Much of the open source work is volunteered. For example, someone might come across a bug in a project they use and submit a quick fix, or they might enjoy tinkering with an open source project in their spare time.
```
Comment (smashwilson) [inline _articles/getting-paid.md:20]: This still doesn't read quite right to me. How about:

```suggestion
Much of the work of open source is volunteer. For example, someone might come across a bug in a project they use and submit a quick fix, or they might enjoy tinkering with an open source project in their spare time.
```

File: _articles/how-to-contribute.md [hunk 3]
```diff
@@ -494,7 +494,7 @@ If the project is on GitHub, here's how to submit a pull request:
 * **Reference any relevant issues** or supporting documentation in your PR (for example, "Closes #37.")
 * **Include screenshots of the before and after** if your changes include differences in HTML/CSS. Drag and drop the images into the body of your pull request.
 * **Test your changes!** Run your changes against any existing tests if they exist and create new ones when needed. Whether tests exist or not, make sure your changes don't break the existing project.
-* **Contribute in the style of the project** to the best of your abilities. This may mean using indents, semi-colons or comments differently than you would in your own repository, but makes it easier for the maintainer to merge, others to understand and maintain in the future.
+* **Contribute to the style of the project** to the best of your abilities. This may mean using indents, semi-colons or comments differently than you would in your own repository, but makes it easier for the maintainer to merge, others to understand and maintain in the future.
```
Comment (smashwilson) [inline _articles/how-to-contribute.md:497]: This one I prefer in its original wording. "Contribute _to_ the style" has a different meaning than "contribute _in_ the style"; the intent here is, "when making a contribution to someone else's repository, try to adhere to the conventions and aesthetics of that repository, even when you have different preferences in your own work," which the original captures a little more clearly imo.

File: docs/styleguide.md [hunk 4]
```diff
@@ -2,7 +2,7 @@
 
 From the GitHub Manual of Style, which this style guide inherits from:
 
-> Words are an important part of how software works. Just as we have a style guide for our code, we have a style guide for our tone and our voice. Even though there may be dozens of people creating a product, it should still sound like we speak in one consistent voice.
+> Words are an important part of how the software works. Just as we have a style guide for our code, we have a style guide for our tone and our voice. Even though there may be dozens of people creating a product, it should still sound like we speak in one consistent voice.
```
Comment (smashwilson) [inline docs/styleguide.md:5]: This one is a direct quote from a different guide, so we probably shouldn't reword it 😄

**Review by smashwilson — CHANGES_REQUESTED** [review]
Thanks! I like almost all of these edits - there are two places I've commented where I believe we'd prefer the original, but all of the others are 👍🏻 .

File: _articles/how-to-contribute.md (hunk 3 above)
Comment (vkay11) [reply _articles/how-to-contribute.md:497]: Yeah, **contribute to** sounds better. My bad!

File: docs/styleguide.md (hunk 4 above)
Comment (vkay11) [reply docs/styleguide.md:5]: Ah didn't know about that... thanks for the info!

File: _articles/getting-paid.md (hunk 2 above)
Comment (vkay11) [reply _articles/getting-paid.md:20]: How about, "_Much of the open source work is voluntary_" or "_Much of the work of open source is voluntary_"?

File: _articles/best-practices.md (hunk 1 above)
Comment (vkay11) [reply _articles/best-practices.md:50]: Yeah instead of a half-assed sounds better. Thanks, mate

File: _articles/getting-paid.md (hunk 2 above)
Comment (vkay11) [reply _articles/getting-paid.md:20]: Okay, then I'll reflect on this change. Thanks for helping!

**Review by smashwilson — APPROVED** [review]

**Comment by smashwilson** [comment]
Thank you! 🙌🏻

**Comment by vkay11** [comment]
> 
> 
> Thank you! 🙌🏻

I should be the one thanking you :-)

File: _articles/getting-paid.md (hunk 2 above)
Comment (ZRDK) [reply _articles/getting-paid.md]: I,'m new

**Pull request merged**